src/control/cmd/dmg/main.go

```diff
@@ -107,12 +122,16 @@ func appSetup(log logging.Logger, opts *cliOptions, conns client.Connect) error
 	}
 	conns.SetTransportConfig(config.TransportConfig)
 
-	ok, out := hasConns(conns.ConnectClients(config.HostList))
-	if !ok {
-		return errors.New(out) // no active connections
+	active, inactive, err := checkConns(conns.ConnectClients(config.HostList))
+	if err != nil {
+		return err
+	}
+	if len(active) == 0 {
+		return errors.New("no active connections")
 	}
 
-	log.Info(out)
+	log.Infof("Active: %s", strings.Join(active, ","))
+	log.Debugf("Inctive: %v", inactive)
```

Comment:
my thinking was that Inactive might not be wanted at info level to avoid clutter, these are compressed and ranges mapped to error message keys, this just prints that Map